x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.test.ts

```diff
@@ -730,7 +382,7 @@ describe('build_exceptions_query', () => {
         lists: payload,
         exclude,
       });
-      const expectedQuery = 'not b:("value-1" or "value-2") and c:"value-3"';
+      const expectedQuery = 'not (b:("value-1" or "value-2") and not c:"value-3")';
```

Comment:
With the `not` in front and parentheses around the whole expression, shouldn't they be joined by ORs? i.e. `const expectedQuery = 'not (b:("value-1" or "value-2") or not c:"value-3")';`

Reply:
I still need to update the last few examples in the description, but with the first ones there, do the results appear to be what is expected?

Reply:
So you're saying `query & !((ex1 & ex2) || (ex1 ex2))` === `query & !(ex1 || ex2) & !(ex1 || ex2)`? But then would that not `and` the entries?

Reply:
When you distribute the `not` across expressions that are ORd together, the inside of each expression isn't affected - so `query & !((ex1 && ex2) || (ex3 && ex4))` === `query && !(ex1 && ex2) && !(ex3 && ex4)`

Reply:
Nevermind. Chatted with Pedro offline. Thanks so much for catching this. I think it was just luck that the examples worked. Updating now!

Reply:
Just to note: We found the previous logic was actually as intended - reverted.